base_agent/dialogue_manager.py

```diff
@@ -38,30 +36,26 @@ class DialogueManager(object):
         agent: a droidlet agent
         model: a (perhaps ML) model used by and the model used for manager.
     """
-    def __init__(self, agent, model):
+
+    def __init__(
+        self, agent, dialogue_object_classes, opts, semantic_parsing_model_wrapper, model=None
```

Comment:
what does the model do?  I would have expected only model_wrapper

Reply:
This `model` was from before so I preserved it. from documentation: a (perhaps ML) model used by and the model used for manager.

Reply:
ok, so lets torch this one then, if someone wants to update the step with a model they can subclass

Reply:
Sounds good! I can do that. Wanted to keep it around for you to take a look at.